docs/source/software/systemwide/matlab.md

```diff
@@ -61,95 +61,372 @@ Use the following procedure if you need to install a MATLAB toolbox:
+1. Select **MATLAB version R2023a** or newer, and set the time for the length of your job, the number of cpus to be 8, and the memory to at least 16 GB. Click **Launch**.
 
 1. If necessary, adjust the **Compression** and **Image Quality**, and then click **Launch MATLAB**.
 
-The following commands `parpool(feature('numcores'))` will start the Parallel Server with the number of cores that are allocated in the MATLAB Open OnDemand job (`feature('numcores')`). You can inspect the Parallel Server and the state of it with the command `parcluster`.
+1. Once MATLAB is open, click on the Home tab, click Parallel > Discover Clusters... to discover the profile. Note, this is valid for R2023a and newer. Alternatively, you can create a cluster profile to run parallel jobs by running `configCluster` in the Command Window. Note: `configCluster` should only be called once on the HPC.
 
-:::{code-block} matlab
-#with parpool
-parpool(feature('numcores'))
+Now MATLAB is configured to have jobs submitted to the HPC cluster and not run in the current session.
 
-#with parcluster
-parcluster
-:::
+### Installation and Configuration of MATLAB on a Local Machine
+MATLAB can be configured on your local machine to submit jobs to the HPC cluster. To complete this, you will need to download the following directory from Github: [MATLAB Desktop Parallel Toolbox Setup](https://github.com/northeastern-rc/matlab-configuration-scripts).
 
-### Using parcluster example
+In your desktop MATLAB instance, run:
 
-:::{note}
-For this example, please select 6 hours, 65 cpus, and 130 GB of memory.
-:::
+```{code-block} matlab
+>> userpath
+```
+
+and move the compressed directory you downloaded from Github to that location and unzip the directory. To configure MATLAB to run parallel jobs on the cluster, run the following in the Command Window:
+
+```{code-block} matlab
+>> configCluster
+```
+
+```{note} 
+This only needs to be called once per version of MATLAB you are using to run jobs on the HPC cluster.
+```
+
+Submitting jobs to the HPC cluster will require SSH credentials and you will be prompted for your username and password or a private SSH key. The username and/or location of the private key will be saved for future session in MATLAB.
+
+Jobs will now default to running on the HPC cluster rather than submitting to your local machine.
+
+To submit jobs to your local machine instead of the HPC cluster, run the following:
+
+```{code-block} matlab
+>> % Get a handle to the local resources
+>> c = parcluster('local');
+```
+
+### Configuring Jobs
+Prior to submitting jobs to the HPC, various parameters can be assigned and adjusted for your job such as partition, email, job walltime, etc.
+
+To get a handle on the cluster and the current configurations please run the following command in the Command Window:
+
+```{code-block} matlab
+>> % Get a handle to the cluster
+>> c = parcluster;
```

Comment:
I get this error here. 

Warning: X does not support locale en_US.UTF-8
>> c=parcluster;
Error using parcluster
The JobStorageLocation of a cluster must be an accessible folder on the current machine's filesystem. The
value supplied (/home/s.caplins/.matlab/generic_cluster_jobs/discovery) is not a valid folder.

Reply:
I figured out my error. I skipped the command "configCluster" because the heading above it says on your desktop, which I thought meant local machine.